README.md

```diff
@@ -0,0 +1,38 @@
+# Operator for OSBuilder
+
+This repo has the operator which provides a K8S API for building [OSBuild](https://www.osbuild.org/) images.
+
+## Installation
+
+### Requirements
+
+- `osbuild >= 26`
+- `go >= 1.16`
+
+## Getting started
+
+### Deployment
+
+- Build and push image:
+
+    `IMG=<image repository and tag> make docker-build docker-push`
+
+    for example: `IMG=quay.io/project-flotta/osbuild-operator:latest make docker-build docker-push`
+- Deploy the operator:
+
+    `IMG=<image repository and tag> make deploy`
+
+    for example: `IMG=quay.io/project-flotta/osbuild-operator:latest make deploy`
+
+In the `osbuild-operator-system` namespace an operator should be running:
+```
+→ oc get pods -n osbuild-operator-system
+NAME                                                  READY   STATUS    RESTARTS   AGE
+osbuild-operator-controller-manager-54f9fdbff-85hfj   2/2     Running   0          2m47s
+```
+
+### Run
+
+- Run the operator locally (outside the cluster):
```

Comment:
also mention that `install` applies project's CRDs on the cluster, or split into:
```
make install
```
and
```
make run
```

Reply:
done